lib/download_test.go

```diff
@@ -366,7 +366,7 @@ func TestDownloadProductFromURL_invalid_public_key(t *testing.T) {
 			ExecutableName: "myprod",
 			ArchivePrefix:  "my_product_download_",
 			PublicKeyId:    downloadProductTestConfig.GpgFingerprint,
-			PublicKeyUrl:   mockServer.URL + "/testproduct/gpg-key.txt",
+			PublicKeyURLs:  []string{mockServer.URL + "/testproduct/gpg-key.txt"},
```

Comment:
Could porbably do witgh a dedicated test to ensure that an invalid URL falls back to the secondary one